website/api/controllers/android-proxy/create-android-enterprise.js

```diff
@@ -93,7 +93,13 @@ module.exports = {
 
       // [?]: https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics/getIamPolicy
       // Retrieve the IAM policy for the created pubsub topic.
-      let newPubSubTopicIamPolicy = await sails.helpers.flow.build(async ()=>{
+      let tryCount = 0;
+      let newPubSubTopicIamPolicy = await sails.helpers.flow.build(async () => {
+        if (tryCount > 0) {
+          // Wait a bit before retrying
+          await new Promise(r => setTimeout(r, 1000));
```

Comment:
Can you replace this with the flow.pause helper?
```suggestion
          await sails.helpers.flow.pause(1000);
```